hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/resource/RestrictedPatientSearchResource.kt

```diff
@@ -38,7 +37,11 @@ class RestrictedPatientSearchResource(private val restrictedPatientSearchService
       example = "[prisonerNumber,firstName,aliases.firstName,currentIncentive.level.code]",
     )
     responseFields: List<String>? = null,
-    @ParameterObject @PageableDefault
-    pageable: Pageable,
-  ) = restrictedPatientSearchService.findBySearchCriteria(searchCriteria, pageable, responseFields)
+    @RequestParam(value = "page", defaultValue = "0")
```

Comment:
started off with `@ParameterObject` here, but couldn't get it to play nicely if the parameter is specified but null e.g. `?page=`.